config/initializers/devise.rb

```diff
@@ -259,7 +259,8 @@
   # ==> OmniAuth
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
-  config.omniauth :yandex, ENV['YANDEX_KEY'], ENV['YANDEX_SECRET']
+
+  config.omniauth :discord, ENV['DISCORD_CLIENT_ID'], ENV['DISCORD_CLIENT_SECRET'], scope: ENV['DISCORD_SCOPE']
```

Comment:
all envs to .env.example please

Reply:
do I need to give them test values?

Reply:
you can leave them empty, just collect all envs.

Reply:
is it ok?
```
DISCORD_CLIENT_ID='test_client_id'
DISCORD_CLIENT_SECRET='test_client_secret'
DISCORD_SCOPE='email identify guilds'
DISCORD_GUILD_ID='test_guild_id'
DISCORD_GUILDS_URL='https://discord.com/api/users/@me/guilds'
```

Reply:
yeap! looks ok